src/fvm_multicell.hpp

```diff
@@ -223,12 +223,30 @@ class fvm_multicell {
 
     std::vector<std::pair<const array fvm_multicell::*, size_type>> probes_;
 
+    struct segment_cv_range {
```

Comment:
add comments that explain why and how each bit of information herein is needed/used.